.github/workflows/dev-backup.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       AWS_DEFAULT_REGION: 'us-east-1'
-      ALGOLIA_INDEX: 'asap-hub_research_outputs_dev'
+      ALGOLIA_INDEX: 'asap-hub_dev'
```

Comment:
this should be also changed with feature flag

Reply:
@niezgoda thats only for the creation of the dev backup. There is no way to change the CI script with a feature flag, right?

Reply:
nope, you could use SSM.
@peterstarling - shall we change it or use SSM?